### DISCUSSION TITLE
dns error: failed to lookup address information: T

### DISCUSSION THREAD

**Discussion by 866666** [messense/aliyundrive-webdav#384] [Q&A] [answered]
### 问题描述

0m refresh token failed, will wait and retry [3merror[0m[2m=[0merror sending request for url (https://websv.aliyundrive.com/token/refresh): error trying to connect: dns error: failed to lookup address information: Try again

### 重现步骤

0m refresh token failed, will wait and retry [3merror[0m[2m=[0merror sending request for url (https://websv.aliyundrive.com/token/refresh): error trying to connect: dns error: failed to lookup address information: Try again

### 版本

最新版

### 运行平台

docker

### 日志

0m refresh token failed, will wait and retry [3merror[0m[2m=[0merror sending request for url (https://websv.aliyundrive.com/token/refresh): error trying to connect: dns error: failed to lookup address information: Try again

**Comment by messense** [comment, marked as answer]
学会搜索：
https://github.com/messense/aliyundrive-webdav/discussions?discussions_q=dns+error
https://github.com/messense/aliyundrive-webdav/issues?q=is%3Aissue+is%3Aclosed+dns+error

**Comment by 866666** [comment]
您发给我的邮件已经被收到

**Comment by xinmans** [comment]
try below to workaround

echo '101.200.125.2 websv.aliyundrive.com' >> /etc/hosts

**Reply by wjs876046992** [reply]
> try below to workaround
> 
> echo '101.200.125.2 websv.aliyundrive.com' >> /etc/hosts

still not work.

**Comment by 866666** [comment]
您发给我的邮件已经被收到
